components/alert-button/alert.css

```diff
@@ -0,0 +1,78 @@
+:root{
+    --blue1:#1e40af;
+    --blue3:#dbeafe;
+    --blue4:#93c5fd;
+    --white-light:#d4d4d4;
+    --grey-light:#a3a3a3;
+    --black-light:#404040;
+    --white-less1:rgb(248, 244, 244);
```

Comment:
Use consistent units for colors: either all hex or all rgb. (Hex is easier in my opinion).